src/spectrum.rs

```diff
@@ -0,0 +1,143 @@
+/// ! Conversions from spectrum intensities to XYZ
+///
+
+use xyz::Xyz;
+use num::{Float,Zero};
+
+
+/// Data from http://www.cvrl.org/cmfs.htm, CIE 1931 2-deg, XYZ CMFs
+/// 360 nm -> 830 nm += 5nm
+const SPECTRUM_TO_XYZ_MAP: [[f32; 3]; 95] = [
```

Comment:
Sure, what I meant was that indexing will always insert a bounds check AFAIK, but the optimization step can then remove it. The question is how accurate it is. I suppose it's nothing to worry about, but I like tuples or structs more for these kind situations. Structs are even nicer because of the named fields. Anyway, there's no need to change if it doesn't matter. :smile: